File: C#/menu_int_hacer_mientras.cs
// Algoritmo menu_int_hacer_mientras
int opcion;
double numero1 = 0, numero2 = 0;
do
{
    // Mostrar menu
    Console.WriteLine("Selecciones una de las opciones");
    Console.WriteLine("1-Suma");
    Console.WriteLine("2-Resta");
    Console.WriteLine("3-Multiplicacion");
    Console.WriteLine("4-Division");
    Console.WriteLine("5-Salir");
    // Leer opcion
    opcion = int.Parse(Console.ReadLine());
    // Validacion de la opcion elejida y obtencion de numeros
    if (opcion >= 1 && opcion <= 4)
    {
        Console.WriteLine("Ingrese el primer numero");
        numero1 = double.Parse(Console.ReadLine());
        Console.WriteLine("Ingrese el segundo numero");
        numero2 = double.Parse(Console.ReadLine());
    }
    // Validacion de opcion elejida y desarrollo de las respectivas operaciones
    switch (opcion)
    {
        case 1:
            double suma = numero1 + numero2;
            Console.WriteLine("Suma: " + suma);
            break;
        case 2:
            double resta = numero1 - numero2;
            Console.WriteLine("Resta: " + resta);
            break;
        case 3:
            double producto = numero1 * numero2;
            Console.WriteLine("Multiplicacion: " + producto);
            break;
        case 4:
            double division = numero1 / numero2;
            Console.WriteLine("Division: " + division);
            break;
        case 5:
            Console.WriteLine("Adiós");
            break;
        default:
            Console.WriteLine("ERROR: Igrese una opcion valida");
            break;
    }
} while (!(opcion >= 1 && opcion <= 5));

File: Python/menu_int_hacer_mientras.py
# Algoritmo menu_int_hacer_mientras
while True:
    # Mostrar menu
    print("Selecciones una de las opciones")
    print("1-Suma")
    print("2-Resta")
    print("3-Multiplicacion")
    print("4-Division")
    print("5-Salir")
    # Leer opcion
    opcion = int(input())
    # Validacion de la opcion elejida y obtencion de numeros
    if opcion >= 1 and opcion <= 4:
        print("Ingrese el primer numero")
        numero1 = float(input())
        print("Ingrese el segundo numero")
        numero2 = float(input())

    # Validacion de opcion elejida y desarrollo de las respectivas operaciones
    match opcion:
        case 1:
            suma = numero1 + numero2
            print("Suma: ", suma)
        case 2:
            resta = numero1 - numero2
            print("Resta: ", resta)
        case 3:
            producto = numero1 * numero2
            print("Multiplicacion: ", producto)
        case 4:
            division = numero1 / numero2
            print("Division: ", division)
        case 5:
            print("Adiós")
        case _:
            print("ERROR: Igrese una opcion valida")

    if opcion >= 1 and opcion <= 5:
        break

